interfaces/Portal/src/assets/i18n/en.json

```diff
@@ -512,6 +483,29 @@
           "waiting-message": "Still waiting for status update on this transaction. Refresh the page for the latest status."
         }
       },
+      "tab": {
+        "dashboard": {
+          "label": "Monitoring"
+        },
+        "overview": {
+          "label": "Overview"
+        },
+        "payment": {
+          "label": "Payment"
+        },
+        "people-affected": {
+          "actionText-disabled-registrations": "Registrations are now disabled for this program",
+          "actionText-enabled-registrations": "Registrations are now enabled for this program",
+          "btnText": "Move to: Payment",
+          "btnText-no-validation": "Move to: Payment",
```

Comment:
Are these labels still being used? The `PhaseNext`-component is gone now, right?